roles/dcos_agent_windows/tasks/dcos_install.yml

```diff
@@ -5,19 +5,11 @@
 
 - name: Windows Installation | Download dcos_install.ps1
   win_get_url:
-    url: "{{ dcos['config']['bootstrap_url'] }}/{{ dcos['version'] }}/genconf_win/serve/prerequisites/dcos_install.ps1"
+    url: "{{ dcos['config']['bootstrap_url'] }}/{{ dcos['version'] }}/genconf/serve/windows/prerequisites/dcos_install.ps1"
     dest: "{{ base_windows_dir }}\\dcos_install.ps1"
 
 - name: Windows Installation | DC/OS install
   when: (ansible_local is not defined) or (ansible_local.dcos_installation is not defined) or (ansible_local.dcos_installation['dcos-image-commit'] is not defined)
   block:
     - name: Windows Installation | Run DC/OS agent installation
       win_shell: "{{ base_windows_dir }}\\dcos_install.ps1 '{{ dcos['config']['bootstrap_url'] }}' '{{ dcos['version'] }}' '{{ groups['masters'] | join(',') }}'"
-      register: install_shell_result
-      notify: reboot the host
```

Comment:
so there is no need for reboot anymore? Does that mean the handler could also be deleted?

Reply:
From the windows node package installation perspective:

- I recall in the past `vcredist` required a restart. 
- But due to different way of installation this seems not any longer to be required.

Reply:
ok just wanted to make sure this is not an accidental remove

Reply:
I will remove handler now